_episodes/01-introduction.md

```diff
@@ -1,15 +1,116 @@
 
+![figure showing ESMValTool architecture]({{ page.root }}/fig/esmvaltool_architecture.png)
+
+> ## Discussion: (dis)advantages of this approach
+>
+> Discuss or think about the pros and cons of this architecture for a moment. Then unfold the box below to see our answers.
+>
+>
+> > ## See our thoughts
+> >
+> > - Streamlining common preprocessing steps ensures consistency in the algorithms being used and the way they are executed. This facilitates comparison and reproducibility.
+> > - Provenance and citation information can be tracked through the entire workflow.
+> > - The core builds upon the [iris](https://scitools.org.uk/iris/docs/latest/) package, which is quite strict in order to comply with CF conventions and minimize unexpected results.
+> > - Recipes are easy to read and share.
+> > - A collection of recipes and diagnostic scripts is shipped with ESMValTool, ready for re-use. Everyone can add to this collection.
+> > - The recipe format takes some getting used to and may be a bit less flexible then working on the datasets directly.
+> > - Features or dataset support that are missing from ESMValCore can be a limiting factor.
+> {: .solution}
+{: .discussion}
+
+## Community
+
+ESMValTool is built and maintained by an active community of scientists and engineers. Many of the interactions take place on GitHub. Here, we briefly introduce you to some of the most important pages.
+
+> ## Challenge: meet ESMValGroup
+>
+> Browse to [github.com/ESMValGroup](https://github.com/ESMValGroup). This is our 'organization' GitHub page. Have a look around. How many collaborators are there? Do you know any of them? You should see 2 main repositories: ESMValTool and ESMValCore. Clicking on them will take you to there. How many people have contributed to each of them? Can you also find out how many people have contributed to this tutorial?
+>
+> > ## Solution
+> >
+> > At the time of making this lesson, there were 138 members of ESMValGroup. 55 of them contributed to ESMValTool, 48 to ESMValCore. 52 (!) people contributed to this tutorial.
+> {: .solution}
+{: .challenge}
+
+> ## Challenge: issues and pull requests
+>
+> Go back to the repository pages of [ESMValTool](https://github.com/ESMValGroup/ESMValTool) or [ESMValCore](https://github.com/ESMValGroup/ESMValCore). You should see tabs for 'issues' and 'pull requests'. You can use the labels to navigate them a bit more. How many open issues are about enhancements of ESMValTool? And how many bugs have been fixed in ESMValCore? There is also an 'insights' tab, where you can see a summary of recent activity. How many issues have been opened and closed in the past month?
+>
+> > ## Solution
+> >
+> > At the time of making this tutorial, there were 50 ESMValTool issues (the majority) about enhancement and 31 bugs had been fixed in the Core. 13 ESMValTool issues had been closed in the past month, versus 8 opened: overall good progress.
```

Comment:
See comment above about the exact numbers.

Reply:
Maybe we should adjust the half sentence "At the time of making this tutorial, ..." to the identical to the one before "At the time of making this lesson..." or vice versa. For consistency in the nomenclature of the "lesson" or "episode"or "tutorial".